cloud-file-browser-ui-update

CollinEMac opened the pull request:
-Fixes breadcrumb caret issue
-Changes folder icons to a manila color and file icons to white.
-Display file location and last modified date columns.

JIRA link - https://divvyhq.atlassian.net/browse/DIV-7350

Commit:
Remove fontawesome caret

Commit:
Revert "Stop disabling tabs so I can test"

This reverts commit b3f2b87d58c2dda022cb5d1bc792764528f820df.

Commit:
Replace bootstrap caret with ascii greater than symbol

Commit:
Make folder icons a brown color

Commit:
Make file icons white

Commit:
Expose file modified date and location

Commit:
Fix moment import typo

Commit:
Set modifiedDate moment to variable

Commit:
Hide location and date columns again to debug

Commit:
Remove moment.js calls

Commit:
Use toLocaleDateString to format date

Commit:
Display raw modified date

Commit:
Install momentjs with npm

Commit:
Fix modifiedDate declaration

Commit:
Format moment date

nmallare commented:
@ganiserb or @letoss Can you please take a gander at this. Collin ran `npm i moment` and it made a bunch of node_module changes. Is this expected?

nmallare commented on divvy/cloudfilebrowser.js:
```diff
@@ -1067,11 +1071,12 @@ var cloudFileBrowser = (function() {
 
+                    var modifiedDate = moment(objItm.modifiedDate).format('MM/DD/YYYY');
+
                     tableHTML += objItm.name + '</li>' +
```
It might be working checking out what else might be available via the API with regards to metadata.

letoss commented:
@nmallare I don't understand why node_modules have been versioned. This is incorrect.

ganiserb commented:
I'm going to reply from email because my browser crashes every time I try
to load the PR diff...

Unversion node_modules :P

On Tue, 12 Feb 2019 at 15:13, Leandro Moreno <notifications@github.com>
wrote:

> @nmallare <https://github.com/nmallare> I don't understand why
> node_modules have been versioned. This is incorrect.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/DivvyHQ/cloud-file-browser-nodejs/pull/5#issuecomment-462871945>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC_Msefn3H1FTU82dIIOW0-ROMAKBPcZks5vMwRSgaJpZM4a0_ab>
> .
>

CollinEMac commented:
@ganiserb and @letoss, I don't understand what you mean by "versioning" node_modules.

Are you guys saying that I can just update my package.json file without checking in changes to node_modules?

ganiserb commented:
Exactly. Because when Divvy installs this dependency it will read
cloudelements' package_json file and install the dependencies in its file
and then it will install Divvy's dependencies. That's my understanding.

On Tue, 12 Feb 2019 at 16:16, Collin MacDonald <notifications@github.com>
wrote:

> @ganiserb <https://github.com/ganiserb> and @letoss
> <https://github.com/letoss>, I don't understand what you mean by
> "versioning" node_modules.
>
> Are you guys saying that I can just update my package.json file without
> checking in changes to node_modules?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/DivvyHQ/cloud-file-browser-nodejs/pull/5#issuecomment-462895512>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC_MsV58mDW4N66j4riScsGJfqWCCMVKks5vMxMlgaJpZM4a0_ab>
> .
>

Commit:
Revert "Install momentjs with npm"

This reverts commit 6d8839e3936b1c5c909e79245d44c02a8f0b4bb0.

Commit:
Install momentjs

Commit:
Remove disable-element class again

This was part of hide-loading-on-accounts-with-no-cloud-storage-accounts story.
At some point I forgot a rebase or something

nmallare reviewed (approved):
GJ, mang.
